tools/project_gen/project_gen/project.py

```diff
@@ -24,6 +24,8 @@
     "CFM_v21p8p1": "CFM_v0_9_Akatore_mod_21p8p1",
 }
 
+EMPIRICAL_MODEL_CONFIG_VERSION = "21p10"
```

Comment:
Just being difficult, but from a usability perspective, it's nice to pass in the full file path rather than a version that then has to be in a specific folder.